modules/ztp-recommended-cluster-kernel-config.adoc

```diff
@@ -12,17 +12,68 @@ Always use the latest supported real-time kernel version in your cluster. Ensure
 +
 [source,yaml]
 ----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  # if you change this name make sure the 'include' line in TunedPerformancePatch.yaml
```

Comment:
We can omit line 17 to 25, otherwise customer will think that they need to explicitly set those name and annotations under metadata. May be after Metadata we ca

Reply:
removed those lines